src/Xive/Mnemonic/Content/FileContents.cs

```diff
@@ -97,7 +97,7 @@ public void UpdateBytes(string name, byte[] data)
 
         public void UpdateXml(string name, XNode xml)
         {
-            this.UpdateBytes(name, new BytesOf(xml.ToString()).AsBytes());
+            this.UpdateBytes(name, new BytesOf(xml.ToString(), Encoding.UTF8).AsBytes());
```

Comment:
I researched and we should use UTF-16 instead of UTF-8 in our libraries because it is the C# default encoding.